### DISCUSSION TITLE
Collapsible code sections

### DISCUSSION THREAD

**Discussion by rmcchrystal** [posit-dev/positron#4207] [Q&A] [answered]
I have had a look around for previous questions about this and in the wiki (possible I missed it still), but I was wondering if there are any plans to adding collapsible code sections in for Positron?

Not a big deal if not but very handy in RStudio for organising and quickly jumping between things using the section panel.

**Comment by juliasilge** [comment, marked as answer]
By "section" do you mean support for folding and symbols in the outline as outlined in https://github.com/posit-dev/positron/issues/3822 for syntax like this:

```r
# My Beautiful Section Doing Difficult Math ------------------------------

1 + 1
```

Or is this about something else?

**Reply by rmcchrystal** [reply]
Hi Julia, that's right support for folding of sections. Apologies I had missed #3822, already answered. Thanks anyway!
